lib/ask_web/views/channel_view.ex

```diff
@@ -20,7 +20,8 @@ defmodule AskWeb.ChannelView do
       channelBaseUrl: channel.base_url,
       settings: channel.settings,
       patterns: channel.patterns,
-      status_info: channel.status
+      status_info: channel.status,
+      userEmail: channel.user_email
```

Comment:
I'd follow [this pattern](https://github.com/instedd/surveda/blob/01cd890c1f6031a0644044523a2311b43eea84b1/lib/ask_web/views/respondent_group_view.ex#L28) to get the user email from the channel:

```elixir
channel = Repo.get(Channel, 1) |> Repo.preload(:user)
```

With the suggested changes, you'll have 2 `Repo.preload`, one inside the query in the `channel_controller` and the other here. Here, the `Repo.preload` is suggested just for completeness, because the channel will be fetched with its user preloaded (because of the `Repo.preload` in the `channel_controller`. That's OK, it won't be loaded twice because by default Ecto won't preload associations that are already loaded. If this function is called from other place, and the user isn't preloaded by its caller, the line above will do it.

And then, you will be able to access the user email this way: `channel.user.email`